documentation/modules/auxiliary/scanner/http/http_hsts.md

```diff
@@ -0,0 +1,66 @@
+## Vulnerable Application
+
+### Description
+
+This module checks to see whether or not the scanned systems return the HSTS header to enforce HSTS.
+
+## Verification Steps
+
+1. Do: ```use auxiliary/scanner/http/http_hsts```
+2. Do: ```set RHOSTS [IP]```
+3. Do: ```set RPORT [PORT]```
+4. Do: ```run```
+
+## Options
+
+  **Proxies**
```

Comment:
as per comments, you can remove all the options, just leave the h2 header.

Reply:
Fixed this now with latest push.

Reply:
@h00die Can confirm if this is good now? If so will mark this as resolved.